.github/workflows/ci-sage.yml

```diff
@@ -0,0 +1,115 @@
+name: Run Sage CI
+
+## This GitHub Actions workflow provides:
+##
+##  - portability testing, by building and testing this project on many platforms
+##    (Linux variants and Cygwin), each with two configurations (installed packages),
+##
+##  - continuous integration, by building and testing other software
+##    that depends on this project.
+##
+## It runs on every pull request and push of a tag to the GitHub repository.
```

Comment:
Is this statement true?

Below there is:
```

on:
  push:
    tags:
      - '*'
  schedule:
    # run each Wednesday at 01:00
    - cron: '0 1 * * 3'
  workflow_dispatch:
    # Allow to run manually
```
Maybe I misunderstand but I don't think that runs on a PR.

Reply:
> Is this statement true?

No, I forgot to update this boilerplate, will fix